app/javascript/handle-forms.js

```diff
@@ -0,0 +1,21 @@
+import $ from 'jquery'
+
+window.$ = $
+
+$(function() {
+  $('button[type="submit"]').on("click", function(e) {
```

Comment:
We should trigger on a form submit, not a button click, otherwise submitting a form with the enter key won't do anything.

Reply:
true - huge oversight on my part, not very accessible at all!

Reply:
wait the strange thing is though - this works without triggering on the form submit and triggering on the button click and I don't know why

Reply:
[click ](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event) covers Enter key and the form has [implicit submission](https://www.w3.org/TR/2011/WD-html5-20110525/association-of-controls-and-forms.html#implicit-submission) on enter when focused on any non submit button part of the form

Reply:
Fair enough!

Reply:
Does that still handle touch events though? eg on a phone with no fields focused